app.py

```diff
@@ -2540,9 +2540,9 @@ def public_user_page(username):
         sorted_adventure_programs = hedy_content.Adventures(
             g.lang).get_sorted_adventure_programs(all_programs, adventure_names)
 
-        favourite_program = None
+        favorite_program = None
         if 'favourite_program' in user_public_info and user_public_info['favourite_program']:
-            favourite_program = DATABASE.program_by_id(
+            favorite_program = DATABASE.program_by_id(
                 user_public_info['favourite_program'])
```

Comment:
I'm not sure if it makes sense haha @hasan-sh because now in one function there are different translations

Reply:
It's a variable that's also saved as favourite, terrible indeed! It would stil be part of what we could change later on.